MAUI/Themes/HowTo.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: How to customize theming in Syncfusion controls
+description: Learn here about how to switch between light and dark themes in Syncfusion Maui controls and more details.
+platform: maui
+control: General
+documentation: ug
+---
+
+# Switching between light theme and dark theme
+
+To switch between light and dark themes in Syncfusion Maui controls, refer to the following example code.
```

Comment:
Refer to the following example code to switch between light and dark themes in Syncfusion Maui controls.

Reply:
Changes addressed.